client/my-sites/themes/theme-showcase.jsx

```diff
@@ -358,20 +353,10 @@ class ThemeShowcase extends Component {
 		this.scrollToSearchInput();
 	};
 
-	getCollectionHeader = () => {
-		let title;
-		let description;
-
-		switch ( this.props.tier ) {
-			case 'premium':
-				title = THEME_COLLECTIONS.premium.title;
-				description = THEME_COLLECTIONS.premium.description;
-				break;
-			case 'marketplace':
-				title = THEME_COLLECTIONS.partner.title;
-				description = THEME_COLLECTIONS.partner.description;
-				break;
-		}
+	getCollectionViewHeader = () => {
+		const key = ( this.props.tier ?? '' ) + ( this.props.filter ?? '' );
+		const title = THEME_COLLECTIONS[ key ]?.title;
+		const description = THEME_COLLECTIONS[ key ]?.description;
```

Comment:
We can simplify like this:
```jsx
const collection = this.props.tier || this.props.filter;
const { title, description } = THEME_COLLECTIONS[ collection ];
```

Reply:
The suggested code isn't exactly the same, the original code is concatenating tiers and filters to form the key, that way we can have composite collections that include a tier and a category. Like Marketplace Arts themes.

The suggested code on the other hand is taking either the tier or the filter to form the key, which isn't quite what I'm aiming for.

Reply:
Got it, although maybe it makes more sense to do something slightly cleaner then, like `tier-feature` rather than `tierfeature` without separation.

The `const { title, description } = THEME_COLLECTIONS[ key ];` is cleaner, but not a massive difference anyway.

Nothing is a blocker! 🙂

Reply:
Sure, I'll change it to use destructuring and add a separator to the parts.

Reply:
Updated.